include/deal.II/lac/affine_constraints.h

```diff
@@ -246,9 +246,10 @@ namespace internal
                        const size_type index_in_constraint) const;
 
       /**
-       * Return whether there is one row with indirect contributions (i.e.,
-       * there has been at least one constraint with non-trivial
-       * ConstraintLine).
+       * Return whether this object stores a constraint in which one degree
+       * of freedom is constrained against another degree of freedom (as
+       * opposed to having only constraints of the form $U_i=42$ where
```

Comment:
typically we list contraints as `u_i ...`, here you capitalize U. Was this on purpose?

Reply:
I switched it to x_i since that's what the documentation of the `AffineConstraints` class uses.